r2-navigator-swift/Toolkit/PaginationView.swift

```diff
@@ -177,6 +181,14 @@ final class PaginationView: UIView, Loggable {
             ? scrollView.contentSize.height - (CGFloat(index + 1) * scrollView.bounds.height)
             : scrollView.bounds.height * CGFloat(index)
     }
+    
+    // finds the index within pageNumbers where the spread's index occurs, adjusted for the minimum page number
+    private func indexForTrimmedPage(_ index: Int, minPage: Int) -> Int {
```

Comment:
Isn't there another place where you make this public? Or am I confusing this with a different thing

Reply:
Confusing with something else